packages/terra-tag/src/Tag.scss

```diff
@@ -0,0 +1,71 @@
+
+:local {
+  .tag {
+    border-radius: 3px;
+    border-style: var(--terra-tag-border-style, solid);
+    border-width: 1px;
+    cursor: pointer;
+    display: inline-flex;
+    font-size: var(--terra-tag-font-size, 14px);
+    font-weight: var(--terra-tag-font-weight, normal);
+    line-height: var(--terra-tag-line-height, 1.14285rem);
+    margin-bottom: var(--terra-tag-margin-bottom, 0.357rem);
+    margin-left: var(--terra-tag-margin-left, 0);
```

Comment:
Since the margin is 0, I believe this can be written as:
```
margin-left: var(--terra-tag-margin-left);
```
Unless of course, some other base style is setting a left margin which you are overriding.

Reply:
Yeah, Brett had mentioned it [here](https://github.com/cerner/terra-core/pull/1288/files/4fd77d0eb6d0b341862d0dada1b4abd23f4e2618) that we do not need fallback values if default theme does not need.

Reply:
Fixed it.